packages/authgear-react-native/index.d.ts

```diff
@@ -85,36 +127,44 @@ export declare class OAuthError extends AuthgearError {
 }
 
 /**
+ * The path of the page in Authgear.
+ *
  * @public
  */
 export declare enum Page {
+  /**
+   * The path of the settings page in Authgear.
+   */
   Settings = "/settings",
+  /**
+   * The path of the indenties page in Authgear.
+   */
   Identities = "/settings/identities",
 }
 
 /**
- * PreAuthenticatedURLDeviceSecretNotFoundError
+ * PreAuthenticatedURLDeviceSecretNotFoundError occured when device secret is not found in the storage.

```

Comment:
@louischan-oursky @tung2744 , when will this happen?

Reply:
> Device secret is not found. This may happen if the "Pre-authenticated URL" feature is not enabled when the user logged in this session. Ask the user to re-login to enable this feature.